TEST_REPORTING.md

```diff
@@ -0,0 +1,73 @@
+# Aries Agent Test Harness: Advanced Test Reporting 
+## Contents<!-- omit in toc -->
+-  [Default Reporting & Output](#default-reporting-&-output)
+-  [Allure Integration](#allure-integration)
+--  [Using Allure with manage Script](#using-allure-with-manage-script)
+--  [Using Allure at the Command Line](#using-allure-at-the-command-line)
+--  [Using Allure in CI](#using-allure-in-ci)
+-  [Other Reporting Options](#other-reporting-options)
+-  [References](#references)
+
+## Default Reporting & Output
+
+The Aries Agent Test Harness(AATH), in utilizing the [Behave](https://behave.readthedocs.io/en/stable/index.html) test engine, has default test output that specifically shows passed/failed steps along with a test/feature summary after an execution run. This output can of course be piped to a file as the user sees fit.  
+
+The need will arise to give more formal and graphic reporting along with keeping historical trends of test executions under a certain configuration. The AATH integrates with the Allure Reporting framework to fulfill this requirement.  
+
+## Allure Integration
+The AATH utilizes Allure in as much as Behave and Allure integrate. See [Behave with Allure](https://docs.qameta.io/allure/#_behave) for details.
+
+### Using Allure with manage Script
+The test execution container that is ramped up with the manage script gets the Allure framework installed for use inside the continer. So to execute the tests and have Allure generated report files, use the `-r allure` option on the `manage` script.
+```
+cd aries-agent-test-harness
+./manage run -d acapy -r allure -t @AcceptanceTest -t ~@wip
+```
+ Running locally and not in a build pipeline/continuous integration system, you will need to install the allure framework in order to generate and display the html report. 
+```
+$ pip install allure-behave
+```
+To generate the html report and start an allure report server. 
+```
+cd aries-test-harness
+allure generate --clean ./reports
```

Comment:
I get "allure:  command not found"

Is there an allure command line we also need to install?

Reply:
https://docs.qameta.io/allure/#_installing_a_commandline